### ISSUE TITLE
unquote-splicing causes runtime error

### ISSUE THREAD

**Issue by dougo** [dyoo/whalesong#21]
This program:

```
`(0 ,@(list 1 2) 4)
```

sucessfully compiles and builds, but viewing the page produces this Javascript error:

```
TypeError: Cannot read property 'namespace' of undefined
```

**Comment by dyoo** [comment]
Investigated: ok, we're missing some bindings from qq-and-or.rkt.  Will have this fixed soon.

I should be able to take the definition of qq-and-or.rkt directly, since it's written in #%kernel (hopefully).

**Comment by dyoo** [comment]
Should be fixed in bf5698c0fa0acdc08fb36cbe6db99ac413d798b9.  Closing issue.
